spotify-conversion-next/package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-config-next": "^13.1.6",
     "framer-motion": "^8.5.5",
     "next": "^13.1.6",
+    "next-auth": "^4.20.1",
```

Comment:
A new package that will help us answer this question:

"How do I keep the refresh/access token persisting through the whole site? Like sure, we got it from a response... but what if we wanted it to be passed from page to page in our **session**?"